Day-I-Social-Card/social-card/src/index.js

```diff
@@ -3,4 +3,6 @@ import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
 
+
+
 ReactDOM.render(<App />, document.getElementById('root'));
```

Comment:
Right here is the only time you need the `ReactDOM.render(<App />, document.getElementById('root'));`